awswrangler/redshift.py

```diff
@@ -72,20 +74,24 @@ def _get_primary_keys(cursor: "redshift_connector.Cursor", schema: str, table: s
 
 def _does_table_exist(cursor: "redshift_connector.Cursor", schema: Optional[str], table: str) -> bool:
     schema_str = f"TABLE_SCHEMA = '{schema}' AND" if schema else ""
-    cursor.execute(
+    sql = (
```

Comment:
Should we also output something like:
```
_logger.info("Created Redshift table {table}")
```
?

Reply:
Thanks, added more logs into `_create_table`.